contracts/colony/Colony.sol

```diff
@@ -39,25 +48,45 @@ contract Colony is BasicMetaTransaction, Multicall, ColonyStorage, PatriciaTreeP
     return token;
   }
 
-  function annotateTransaction(bytes32 _txHash, string memory _metadata) public always {
+  function annotateTransaction(
+    bytes32 _txHash,
+    string memory _metadata
+  ) public always {
     emit Annotation(msgSender(), _txHash, _metadata);
   }
 
-  function emitDomainReputationReward(uint256 _domainId, address _user, int256 _amount)
-  public stoppable auth
-  {
+  function emitDomainReputationReward(
+    uint256 _domainId,
+    address _user,
+    int256 _amount
+  ) public stoppable auth {
     require(_amount > 0, "colony-reward-must-be-positive");
     require(domainExists(_domainId), "colony-domain-does-not-exist");
-    IColonyNetwork(colonyNetworkAddress).appendReputationUpdateLog(_user, _amount, domains[_domainId].skillId);
-
-    emit ArbitraryReputationUpdate(msgSender(), _user, domains[_domainId].skillId, _amount);
-  }
-
-  function emitSkillReputationReward(uint256 _skillId, address _user, int256 _amount)
-  public stoppable auth validGlobalOrLocalSkill(_skillId)
-  {
+    IColonyNetwork(colonyNetworkAddress).appendReputationUpdateLog(
+      _user,
+      _amount,
+      domains[_domainId].skillId
+    );
+
+    emit ArbitraryReputationUpdate(
+      msgSender(),
+      _user,
+      domains[_domainId].skillId,
+      _amount
+    );
+  }
+
+  function emitSkillReputationReward(
+    uint256 _skillId,
+    address _user,
+    int256 _amount
+  ) public stoppable auth validGlobalOrLocalSkill(_skillId) {
     require(_amount > 0, "colony-reward-must-be-positive");
-    IColonyNetwork(colonyNetworkAddress).appendReputationUpdateLog(_user, _amount, _skillId);
+    IColonyNetwork(colonyNetworkAddress).appendReputationUpdateLog(
```

Comment:
I like 80 better than 140, but it does seem to wrap almost every call... how would you feel about 100 as a target?